data/random-teams.ts

```diff
@@ -1381,6 +1406,9 @@ export class RandomTeams {
 				// Limit to one of each species (Species Clause)
 				if (baseFormes[species.baseSpecies]) continue;
 
+				// Prevent unwanted forms that shouldn't be in doubles in doubles
+				if (this.format.gameType !== 'singles' && !species.randomDoubleBattleMoves) continue;
```

Comment:
What does this affect specifically? I don't see why there should be any cases of it.